src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,103 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_SIZE = 10;
+    private Object[] values;
+    private int size;
+
+    public ArrayList() {
+        values = new Object[DEFAULT_SIZE];
+        size = 0;
+    }
+
     @Override
     public void add(T value) {
-
+        if (size == values.length) {
+            getNewSize();
```

Comment:
```suggestion
            grow();
```
if you start your method name with get it would suggest that you would get some value from that method